examples/grpc/README.md

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
I realized we should update our examples to use library instrumentation where possible. This seems like an example of an anti pattern, instrumenting a library yourself.

Reply:
I would love to see an example that uses some library instrumentation!